src/crypto/clu_evp_crypto.c

```diff
@@ -203,18 +242,17 @@ int wolfCLU_evp_crypto(const WOLFSSL_EVP_CIPHER* cphr, char* mode, byte* pwdKey,
 
     if (printOut) {
         int z;
-        WOLFCLU_LOG(WOLFCLU_L0, "salt [%d] :", SALT_SIZE);
-        for (z = 0; z < SALT_SIZE; z++)
-            WOLFCLU_LOG(WOLFCLU_L0, "%02X", salt[z]);
-        WOLFCLU_LOG(WOLFCLU_L0, " ");
-        WOLFCLU_LOG(WOLFCLU_L0, "key  [%d] :", keySz);
+        if (!noSalt) {
+            printf("salt [%d] :", SALT_SIZE);
```

Comment:
Note for future improvement - put `printf` behind a macro in case some platforms want to handle output differently.